aws-api-request.js

```diff
@@ -141,14 +174,21 @@ function createResult(data, res) {
     }
 }
 
-function request(method, path, headers, querystring, data, callback) {
+function request(method, path, headers, querystring, data, retryAttempt, callback) {
     
     let qs = Object.keys(querystring).map(k => `${k}=${encodeURIComponent(querystring[k])}`).join('&');
     path += '?' + qs;
     let hostname = headers.Host;
     delete headers.Host;
     headers['Content-Length'] = data.length;
     const port = 443;
+
+    //Test of exponential backoff, use at least 3 attempts for every operation ...
+    if (retryAttempt < 3) {
+    //    callback(null, { statusCode: 400, data : { Error : { Code : 'Throttling'}}});
+    //    return;
+    }
+
```

Comment:
At first look, this is redundant and we can remove it -- the above should take care of it all.

Is that correct?